src/components/ActionsMenu/ActionsMenu.e2e.ts

```diff
@@ -0,0 +1,30 @@
+import { browser } from 'protractor';
+import {
+  createStoryUrl,
+  waitForVisibilityOf,
+} from 'wix-ui-test-utils/protractor';
+import { actionsMenuTestkitFactory } from '../../testkit/protractor';
+
+/**
+ * For tests containing interactions.
+ * Can be removed if not used.
```

Comment:
This file can be removed if there are no special cases to test, that require a real dom.
Just to see that it is rendered in a browser is done in the visual tests.